rllib/tuned_examples/a2c/cartpole_a2c.py

```diff
@@ -0,0 +1,11 @@
+from ray.rllib.algorithms.a2c import A2CConfig
```

Comment:
Can't believe this is actually the shortest and best looking (self-documented) example file of all the different extension types! :D

Reply:
yeah, right :)